### PR TITLE
Slim down response logger a little

### PR DESCRIPTION
**Pull request by cyx**

### CODE REVIEW

**Comment by cyx** [comment]
@freeformz finally got around to creating the PR for this. Thanks a lot! :+1:

**Comment by freeformz** [comment]
FWIW: At first I thought I could do a lot more, but turns out ... nope.

**Comment by cyx** [comment]
Nah it's still good! Reminded me to do the same style for interfaces as
it's a pattern that's recurring in the code base.

File: util/response_logger.go [hunk 1]
```diff
@@ -6,41 +6,30 @@ import (
...
-func NewResponseLogger(w http.ResponseWriter) *responseLogger {
-	return &responseLogger{w: w, status: http.StatusOK}
+func NewResponseLogger(w http.ResponseWriter, requestID string) *responseLogger {
+	return &responseLogger{ResponseWriter: w, status: http.StatusOK, requestID: requestID}
 }
 
 type responseLogger struct {
-	w      http.ResponseWriter
-	status int
-	size   int
```
Comment (tt) [inline util/response_logger.go]: Why didn't we use this? Did it get lost in some refactoring?

**Pull request merged**